msr-flows/Notes/type_1_flow.ipynb

```diff
@@ -0,0 +1,152 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Linear Growth Approximation\n",
+    "\n",
+    "#### Only suitable for SS\n",
+    "\n",
+    "The linear growth approximation is an approach which assumes that over some time step X, the growth is approximately linear. This means that the removal constant must be calculated such that the number of atoms at time X is the same as SaltProc at that time. As long as accuracy can be maintained to some position past where the SaltProc value is determined, then this method can be used to reduce the computational time of SaltProc by making a series of small linear approximations.\n",
```

Comment:
Here's some formatting and flow suggestions to enhance clarity. I'm not quite sure if my edits here reflect what you're trying to convey. If they don't, you can ignore this suggestion.
```suggestion
    "The _linear growth approximation_ is an approach which assumes that over some time step X, the growth is approximately linear. This means that the removal constant must be calculated such that the number of atoms at time X is the same as in the SaltProc calculation at time X. As long as accuracy can be maintained to some position past where the SaltProc value is determined, then this method can be used to reduce the computational time of SaltProc by making a series of linear approximations.\n",
```

Reply:
tagging @nsryan2 for another opinion